src/QemuRunner.jl

```diff
@@ -0,0 +1,170 @@
+"""
+    QemuRunner
+
+A `QemuRunner` represents an "execution context", an object that bundles all
+necessary information to run commands within the container that contains
+our crossbuild environment.  Use `run()` to actually run commands within the
+`QemuRunner`, and `runshell()` as a quick way to get an interactive shell
+within the crossbuild environment.
+"""
+type QemuRunner <: Runner
+    qemu_cmd::Cmd
+    append_line::String
+    sandbox_cmd::Cmd
+    platform::Platform
+end
+
+qemu_path() = "/Users/kfischer/Projects/qemu-hvf/x86_64-softmmu/qemu-system-x86_64"
+kernel_path() = "/Users/kfischer/Projects/qemu-hvf/x86_64-softmmu/bzImage"
```

Comment:
So ideally both of these will be sourced via `BinaryProvider`, right?  I guess it would be fun to make a `Builder` repository for the kernel itself.....

Reply:
Yes and yes